### ISSUE TITLE
Allow to configure the size of the popup

### ISSUE THREAD

**Issue by mike-tn** [2alin/tabFx#8]
## Description
Current popup size is fixed to 390x420 pixels and could not be enough size to been able to read the tab's title in the result list.

## What to change
Provide a way to customize the size of the popup.

---
_Original request:_

Popup window is rather small and long tab titles doesn't fit.
It would be nice to have a button (somewhere on top) to enlarge popup window (something about 80% of width and height of FF window). 
Then second click must shrink it back to original size. And so on...

P.S. Also please make popup window resizeble, if possible.

**Comment by 2alin** [comment]
what about adding all that overflow content in the tooltip you proposed before?
Although, configurable popup width sounds good as well.

**Comment by mike-tn** [comment]
> what about adding all that overflow content in the tooltip you proposed before?

Yes, this is a solution. (But may be not as good as changable pupup width)
